<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config_dialog</class>
 <widget class="QDialog" name="config_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPRAT Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Binscanner Memory Model:              </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="small_radioButton">
       <property name="text">
        <string>Small</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="medium_radioButton">
       <property name="text">
        <string>Medium</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="large_radioButton">
       <property name="text">
        <string>Large</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="huge_radioButton">
       <property name="text">
        <string>Huge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="auto_radioButton">
       <property name="text">
        <string>Auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>NOTE: If checked, the user must copy blackboard.simulation.xml from the metadata folder to the SAF</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="blackboard_checkBox">
     <property name="text">
      <string>Process  ARTEMIS Blackboard File (blackboard.simulation.xml)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="efi_checkBox">
     <property name="text">
      <string>Process EFI data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="dfi_checkBox">
     <property name="text">
      <string>Process DFI data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="imacs_tlm_checkBox">
     <property name="text">
      <string>Use ONLY IMACS Telemetry Definitions (ignore tlm_cfg.xml file - 'noxml' option)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="gige_checkBox">
     <property name="text">
      <string>Use ONLY gige.bin files (no rs422 or 1553 files)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pcap_checkBox">
     <property name="text">
      <string>Use ONLY PCAP files (requires Wireshark to be installed)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stream_checkBox">
     <property name="text">
      <string>Process STREAM data (archived at HOSC)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="skip_stream_checkBox">
     <property name="text">
      <string>Skip STREAM Processing (data has already been processed with STREAM option)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tlm_files_checkBox">
     <property name="text">
      <string>Generate Telemetry Playback (.dem) Files</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="max_fp_digs_checkBox">
     <property name="text">
      <string>Maximum floating point digits ( %24.16e )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="debug_checkBox">
     <property name="text">
      <string>Enable binscanner DEBUG mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="irs_checkBox">
     <property name="text">
      <string>Perform iRS (iterative Reconstructive Simulation of GNC)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="exe_path_lineEdit">
       <property name="text">
        <string>&lt;default&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="exe_browse_pushButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Executable Path:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Default Plot Program:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="plot_comboBox">
       <item>
        <property name="text">
         <string>gnuplot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>winplot</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>config_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>config_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
